src/pages/[platform]/build-a-backend/add-aws-services/deletion-backup-resources/index.mdx

```diff
@@ -0,0 +1,136 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Deletion protection and Backup resources',
+  description: 'Learn how to enable deletion protection and backup on resources.',
+  platforms: [
+    'javascript',
+    'react-native',
+    'angular',
+    'nextjs',
+    'react',
+    'vue'
+  ],
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+
+<Callout warning>
+
+Deleting a Amplify sandbox with a resource enabled with deletion protection, the deploy process will fail and the resource will need to be manually deleted on the AWS console. 
+
+</Callout>
+
+Using the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/latest/guide/home.html) we can configure Amplify generated resource to enable deletion protection and backups on supported resources. For example, you can use AWS CDK to enable [Point-in-time recovery for DynamoDB tables](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery.html), or use [AWS Backup](https://aws.amazon.com/backup/) as a advanced backup option.
+
+Using underlying CDK construct properties you can modify resource configurations. This allows you to customize backend resources beyond what is offered via the `define*` functions.
+
+In the `amplify/backend.ts` create a `backend` object that exposes resource properties with objects for each of the components passed into the `defineBackend` function.
+
+```ts title="amplify/backend.ts"
+import { defineBackend } from '@aws-amplify/backend';
+import { auth } from './auth/resource';
+import { data } from './data/resource';
+
+const backend = defineBackend({
+  auth,
+  data
+});
+```
+
+## Enabling Deletion protection on a Auth resource
+
+For example, if you would like to enable Deletion protection on a Cognito user pool resource created by Amplify Auth.
+
+```ts title="amplify/backend.ts"
+backend.auth.resources.cfnResources.cfnUserPool.deletionProtection = "ACTIVE";
+```
+
+## Enabling Deletion protection on a Data resource
+
+For example, if you would like to enable Deletion protection on all DynamoDB tables created by GraphQL API.
+
+```ts title="amplify/backend.ts"
+const dataResources = backend.data.resources.cfnResources;
+for (const table of Object.values(dataResources.amplifyDynamoDbTables)) {
```

Comment:
```suggestion
const { amplifyDynamoDbTables } = backend.data.resources.cfnResources;
for (const table of Object.values(amplifyDynamoDbTables)) {
```

for readability

separately, is this located under cfnResources?

Reply:
yes, this is now located under cfnResources: 
https://docs.amplify.aws/gen2/build-a-backend/data/override-resources/